scala-package/examples/src/main/scala/org/apache/mxnetexamples/imclassification/TrainMnist.scala

```diff
@@ -31,6 +31,7 @@ object TrainMnist {
   def getMlp: Symbol = {
     val data = Symbol.Variable("data")
 
+    // val fc1 = Symbol.FullyConnected(name = "relu")()(Map("data" -> data, "act_type" -> "relu"))
```

Comment:
no need to keep it commented, you can remove it completely. change it any of your next PRs. I am going to merge it as is instead of removing this line, I don't want the entire CI suite to run for hour.